.github/workflows/nightly.yml

```diff
@@ -40,34 +51,34 @@ jobs:
       version: "graal/master"
       build-type: "graal-source"
       jdk: "21/ea"
-      build-stats-tag: "gha-linux-qmain-glatest-jdk21ea"
+      build-stats-tag: "gha-linux-graal-qmain-glatest-jdk21ea"
```

Comment:
FWIW `glatest` stands for graal latest and `mlatest` for mandrel latest. I know, not the best choice... :)

Reply:
aah, yeah hard to spot.